docs/installation.rst

```diff
@@ -661,28 +661,29 @@ Installing on Ubuntu Linux or other apt-get--based distributions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The following is designed to obtain a working QMCPACK build on, for example, a
-student laptop, starting from a basic Linux installation with none of
-the developer tools installed. Fortunately, all the required packages
-are available in the default repositories making for a quick
-installation. Note that for convenience we use a generic BLAS. For
-production, a platform-optimized BLAS should be used.
-
+student laptop, starting from a basic Linux installation with none of the
+developer tools installed. Fortunately, all the required packages are available
+in the default repositories making for a quick installation. Note that for
+convenience we use a generic BLAS. A vendor-optimized BLAS is usually faster.
 
 ::
 
   sudo apt-get install cmake g++ openmpi-bin libopenmpi-dev libboost-dev
-  sudo apt-get install libatlas-base-dev liblapack-dev libhdf5-dev libxml2-dev fftw3-dev
-  export CXX=mpiCC
+  sudo apt-get install libopenblas-dev libhdf5-dev libxml2-dev libfftw3-dev
```

Comment:
It is extremely important to install `libopenblas-openmp-dev` instead of `libopenblas-dev`.